### DISCUSSION TITLE
Problem with FindTransferAnchors from a converted h5 file to Seurat Object

### DISCUSSION THREAD

**Discussion by amberdeja** [satijalab/seurat#5317] [General]
Hello, 

I am trying to map a query data set onto a reference. It works fine, when my query dataset comes from a Seurat object but not when I upload an h5 file and convert it to a Seurat object.

This is what I use to convert the data to a Seurat object:
skin244 <- Read10X_h5(filename = "/ filtered_feature_bc_matrix.h5", use.names = T, unique.features = T)
skin244.query <- CreateSeuratObject(counts = skin244$"Gene Expression", project = "skin244")

I get this warning message.
Genome matrix has multiple modalities, returning a list of matrices for this genome
Warning message:
In sparseMatrix(i = indices[] + 1, p = indptr[], x = as.numeric(x = counts[]),  :
  'giveCsparse' has been deprecated; setting 'repr = "T"' for you

Then when I try to get the anchors using this Seurat Object I get this error message:
> skin.anchors1 <- FindTransferAnchors(reference = skin.integrated, query = skin28.query, dims = 1:30, reference.reduction = 'pca')
Projecting cell embeddings
Finding neighborhoods
Finding anchors
	Found 87 anchors
Filtering anchors
Error in FilterAnchors(object = object.pair, assay = assay, slot = slot,  : 
  No features provided and no VariableFeatures computed.

Is there a problem with the conversion of the h5 file to the Seurat Object?

Thank you,
Alejandra

**Comment by ETAOIK** [comment]
@amberdeja I am facing the same traceback, May I ask if you have solved this issue and how to solve it? Any advice is appreciated!
